### PR TITLE
HelpWithAim

### PR DESCRIPTION
**Pull request by SuperNinjaMati**
It changes the aim thingy in game

### CODE REVIEW

File: texturepacks/RemisTexturePack/_meta_json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Remi's texture pack",
+  "author": "Remi",
+  "description": "Such a cool pack!"
```
Comment (itzTheMeow) [inline texturepacks/RemisTexturePack/_meta_json:4]: can you change this to be a bit more descriptive?

**Review by itzTheMeow — CHANGES_REQUESTED** [review]

File: texturepacks/RemisTexturePack/_meta_json (hunk 1 above)
Comment (SuperNinjaMati) [reply texturepacks/RemisTexturePack/_meta_json:4]: wdym a bit more descriptive?

Comment (SuperNinjaMati) [reply texturepacks/RemisTexturePack/_meta_json:4]: what should i do?

Comment (itzTheMeow) [reply texturepacks/RemisTexturePack/_meta_json:4]: describe what the pack actually adds or changes

Comment (SuperNinjaMati) [reply texturepacks/RemisTexturePack/_meta_json:4]: kk

Comment (itzTheMeow) [reply texturepacks/RemisTexturePack/_meta_json:4]: you didnt edit the file

**Comment by itzTheMeow** [comment]
closing this due to inactivity. if you want to re-add your pack you can with a proper description

**Pull request closed without merging**